docs/solana/migration/hotspot-maker.mdx

```diff
@@ -317,10 +317,69 @@ like to use the library, the setup guide can be found here
 
 ---
 
-## Data Credits and Sol for Onboarding
-
-<!-- TODO: We need to create an easy way for makers to get sol and data credits into their accounts -->
-
-```
-
-```
+## Data Credits and SOL for Onboarding
+
+:::info
+
+Maker Wallets will need both `SOL` and `DC` for transactions post Migration.
+
+:::
+
+### Maker Wallet SOL
+
+Maker Wallets will need `SOL` to pay transaction fees and for minting new merkle trees if creating
+double the current amount of Hotspots in their existing fleet.
+
+- For most makers, ~10 SOL is recommended.
+- Makers with large fleets (300k+ Hotspots) ~60 SOL is recommended.
+- For more insight to the amount of SOL needed, check out
+  [https://compressed.app/](https://compressed.app/)
+  - Enter the number of Hotspots to mint in the `How many compressed NFTs do you want to store?`
+    field.
+  - The `Uncompressed equivalent` value is a good approximation, round up when in doubt.
```

Comment:
We might want to leave this calculator out. It's confusing that we are constantly talking about compression NFTs, but then we tell them to ignore that pricing here.